client/state/selectors/get-jetpack-onboarding-pending.js

```diff
@@ -0,0 +1,33 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { reduce } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { JETPACK_ONBOARDING_STEPS as STEPS } from 'jetpack-onboarding/constants';
+import { saveJetpackOnboardingSettings } from 'state/jetpack-onboarding/actions';
+import { getRequest } from 'state/selectors';
+
+export default function getJetpackOnboardingPending( state, siteId, steps ) {
```

Comment:
Do you think we can clarify the name a bit? Because right now it's unclear at first glance what's the pending thing that we're retrieving. Perhaps rephrase the selector to `getJetpackOnboardingPendingSteps`?

Reply:
Sure -- I was trying to be consistent with `getJetpackOnboardingProgress`. Should we consequently rename that to `getJetpackOnboardingCompletedSteps`?

Reply:
Sounds great!

Reply:
#21596